.github/workflows/android-smoke-test.yml

```diff
@@ -19,7 +19,7 @@ on:
 jobs:
   run:
     name: try-${{ inputs.try }}
-    runs-on: macos-latest
+    runs-on: macos-13
```

Comment:
But based on [this comment](https://github.com/actions/runner-images/issues/9472#issuecomment-1984324388) 13 & 14 do not allow the VM within a VM.

Was there a reason the Android tests need to run on macOS?

Reply:
It no longer needs to run on macOS, I am changing it to ubuntu